controllers/home-routes.js

```diff
@@ -48,53 +48,17 @@ router.get('/hero/:id', (req, res) => {
     })
 })
 
+// Direct to login-screen
+router.get('/login', (req, res) => {
+res.render('login')
 
-// //-- GET all KBAs for homepage
-// //TODO:: 02/10/2022 #EP | Build for KBAs
-router.get('/', async (req, res) => {
-    try {
-      //TODO:: 02/10/2022 #EP | Build for KBAs
-      const userData = await User.findAll({ 
-        include: [
-          {
-            // model: KBA,
-            attributes: ['id', 'username'],
-          },
-        ],
-      });
-  
-      //TODO:: 02/10/2022 #EP | Build for KBAs
-      const kbas = kbaData.map((kba) =>
-        kba.get({ plain: true })
-      );
-  
-      res.render('homepage', {
-        kbas,
-        loggedIn: req.session.loggedIn,
-      });
-    } catch (err) {
-      console.log(err);
-      res.status(500)
-      .json({
-        request: {
-            method: req.method,
-            params: req.params,
-            body: req.body,
-            path: "./home-routes",
-        },
-        response: {
-            status: 404,
-            message: "Rquest failure. Page not found."
-           }
-        })
-    }
-});
+})
```

Comment:
Removed this extra route that was from my seed-template still, and replaced it with the **/login** route for Handlebars.js